pyproject.toml

```diff
@@ -14,6 +14,7 @@ fastapi = "^0.62.0"
 uvicorn = "^0.13.1"
 Jinja2 = "^2.11.2"
 aiofiles = "^0.6.0"
+# :index:`fixme`: shouldn't we install a `backend <https://www.encode.io/databases/#installation>`_?
```

Comment:
It would be nice if we could do :index:`fixme:` or :index:`FIXME:` so that these tags are compatible with the usual editor plugins.

Reply:
On VSCode, making that `` :index:`FIXME` `` works. How about that?